### PR TITLE
"Assigned weight to components to make icon visible"

### LINKED ISSUE #3434: Help popup sometimes invisible in Achievements
All stats have a help button, except that one.
I guess it is just waiting for someone to add one.
The help text could be:

> In the app's home, tap the "Nearby" tab to show places around you that need a picture.

### PR DESCRIPTION
**Pull request by isdl03**
**Description (required)**

Fixes #3434

What changes did you make and why?
I have assigned the width of the text and the icon in terms of weight of the layout so that icon and text will get percentage of the layout and icon doesn't become visible.
{
At line 305 I have assigned width=0 
At line 306 I have assigned weight=0.8
At line 316 I have assigned width=0 
At line 317 I have assigned weight=0.1
remaining code is same
}
**Tests performed (required)**
Tested on Pixel with API level 28.

**Screenshots (for UI changes only)**
Before solving  at 320 (smallest width)
https://photos.app.goo.gl/vV4rg4wsHcBU6hdv7
After solving at 320(smallest width) 
https://photos.app.goo.gl/Xa5NCVozzNnN5vkd8

_Note: Please ensure that you have read CONTRIBUTING.md if this is your first pull request._

### CODE REVIEW

File: app/src/main/res/layout/fragment_achievements.xml [hunk 1]
```diff
@@ -6,542 +6,544 @@
...
+              android:layout_width="@dimen/overflow_icon_dimen"
+              android:layout_height="@dimen/overflow_icon_dimen"
+              android:id="@+id/wikidata_edits_icon"
+              app:srcCompat="@drawable/ic_custom_map_marker" />
 
             <TextView
-              style="?android:textAppearanceMedium"
-              android:layout_width="wrap_content"
+              android:layout_width="0dp"
+              android:layout_weight="0.8"
```
Comment (neslihanturan) [inline app/src/main/res/layout/fragment_achievements.xml:303]: hey @isdl03 , thanks for these changes. However this is one of few related lines with the issue. Why all these lines are edited? Can you please make sure just related edits are made?

**Review by neslihanturan — CHANGES_REQUESTED** [review]
Please edit only related lines :)

**Comment by arishta** [comment]
@neslihanturan We have the requested changes. Kindy review them.

File: app/src/main/res/layout/fragment_achievements.xml (hunk 1 above)
Comment (arishta) [reply app/src/main/res/layout/fragment_achievements.xml]: @neslihanturan We have made the requested changes. Kindy review them.

**Review by neslihanturan — CHANGES_REQUESTED** [review]
Hey @isdl03 , thanks for your changes. Even if this fixes the issue, the code causes another UI issue. Here is how it looks on a high resolution device on master:
resim

Here is how it looks on a high resolution device on your branch (Did you see, a gap is added. We don't want to change current behavior on high resolution devices.):
 
resim

**Comment by neslihanturan** [comment]
Seems like this PR is sleeping, feel free to reopen if needed.

**Pull request closed without merging**